### PR TITLE
Automatically generate derived Channel schemas

### PR DESCRIPTION
**Pull request by jakevdp**

### CODE REVIEW

File: tools/generate_schema_interface.py [hunk 1]
```diff
@@ -187,27 +183,67 @@ def print_(*args, **kwargs):
...
+    def write_wrappers(self, path=None):
+        # make sure the path is valid
+        if path is None:
+            path = getpath('..', 'altair', 'schema', '_wrappers')
+        if not os.path.exists(path):
+            os.makedirs(path)
+
+        print("Writing wrappers to {0}".format(path))
+
+        classes = [('PositionChannel', 'PositionChannelDef'),
```
Comment (ellisonbg) [inline tools/generate_schema_interface.py]: Let's create a `CLASS_ALIASES` constant at the top of the file and also include the other aliases that are in the api.

Comment (jakevdp) [inline tools/generate_schema_interface.py]: Great idea – I'll make it a bit more flexible as well.

**Comment by jakevdp** [comment]
Travis says it can't find the branch. Any ideas?

**Comment by jakevdp** [comment]
I think I messed something up by accidentally pushing this branch to upstream. I'm going to try closing this and opening a new PR.

**Pull request closed without merging**